analyze.js

```diff
@@ -104,19 +104,72 @@ async function attachScanFiles(scanGlobs, formData) {
   }
 }
 
+async function validateBranches(branches, config) {
+  const defaultBranch = branches.filter(branch => branch.isDefault)[0].name
+
+  // If target branch is not defined, use the default branch. This also ensures even if
+  // base branch is defined, the default branch is not created as a child of the base branch
+  if (typeof config.targetBranchName == 'undefined') {
```

Comment:
This is an unnecessarily specific check, what if the user sets to `null` or an empty string? In those cases we'd want to treat the param as unset, but rather than checking for each individual case, can just use a "false-ish"/"falsey"  check with e.g. `if (!config.targetBranchName)`